XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -109,99 +171,324 @@
+                <local:ControlExampleSubstitution Key="AreStickyGroupHeadersEnabled" Value="{x:Bind StickySwitch.IsOn, Mode=OneWay}"/>
+            </local:ControlExample.Substitutions>
         </local:ControlExample>
 
-        <local:ControlExample HeaderText="A ListView with Grouped Sticky Headers" XamlSource="ListView\ListViewStickyHeaderSample_xaml.txt">
-            <ListView 
-                ItemsSource="{x:Bind ContactsCVS.View, Mode=OneWay}"
-                ItemTemplate="{StaticResource ContactListViewTemplate}"
-                SelectionMode="Single"
-                ShowsScrollingPlaceholders="True"
-                Height="400" 
-                MinWidth="550"
-                BorderThickness="1"
-                BorderBrush="DimGray">
-                <ListView.GroupStyle>
-                    <GroupStyle >
-                        <GroupStyle.HeaderTemplate>
-                            <DataTemplate x:DataType="local1:GroupInfoList">
-                                <Border AutomationProperties.Name="{x:Bind Key}">
-                                    <TextBlock Text="{x:Bind Key}" Style="{ThemeResource TitleTextBlockStyle}"/>
-                                </Border>
-                            </DataTemplate>
-                        </GroupStyle.HeaderTemplate>
-                    </GroupStyle>
-                </ListView.GroupStyle>
-            </ListView>
+        <local:ControlExample HeaderText="ListView for Messaging or Data Logging">
+            <StackPanel>
+                <TextBlock Margin="0 0 0 15" TextWrapping="Wrap">
+                    This ListView is inverted to grow from the bottom up. It's a good way to display logs or messages.
+                </TextBlock>
+                <ListView
+                    x:Name="InvertedListView"
+                    ItemTemplate="{StaticResource MessageViewTemplate}"
+                    BorderThickness="1"
+                    BorderBrush="DimGray"
+                    Height="400"
+                    SelectionMode="None">
+                    
+                    <ListView.ItemsPanel>
+                        <ItemsPanelTemplate>
+                            <ItemsStackPanel ItemsUpdatingScrollMode="KeepLastItemInView" VerticalAlignment="Bottom"/>
+                        </ItemsPanelTemplate>
+                    </ListView.ItemsPanel>
 
+                    <ListView.ItemContainerStyle>
+                        <Style TargetType="ListViewItem">
+                            <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
+                        </Style>
+                    </ListView.ItemContainerStyle>
+
+                </ListView>
+            </StackPanel>
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
-&lt;CollectionViewSource x:Name="ContactsCVS" IsSourceGrouped="True"/&gt;
+&lt;!-- The ListView itself does not need any new properties added to its definition. --&gt;
```

Comment:
It's using a unique data template, I would add that to the code sample. And maybe remove this comment...? I'm not sure.